README.md

```diff
@@ -13,12 +13,22 @@ docker run -p 80:80 -e PORT=80 -e JWT_SECRET=test uoftboard
 
 Setup needs to be done in order to use the type definitions from the server in our client files.
 
-Make sure that "declaration": true is set in server tsconfig and perform yarn build in server
+Make sure that `"declaration": true` is set in server tsconfig and perform `yarn build` in server dir
 
 Then in the root directory of the project run this:
 
+Windows:
```

Comment:
![but-why-wtf](https://user-images.githubusercontent.com/12144493/142566017-ec0b416a-f3fc-476a-8fe5-497aa0b1288a.gif)


Posix is better

Reply:
Windows 🤣🤣🤣🤣

Reply:
no xbox is better